docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -0,0 +1,137 @@
+# Use Kaito in AKS Arc
+This article describes how to deploy AI models on AKS arc with AI toolchain operator (KAITO). The AI toolchain operator (KAITO) is a managed add-on for all AKS, and it simplifies the experience of running OSS AI models on your AKS clusters. You may follow the workflow below to enable this feature:
+1.	Create a Node pool with GPU
+2.	Deploy KAITO operator
+3.	Deploy AI model
+4.	Validate the deployment
+
+## Supported Models
+Currently KAITO supports models such as Falcon, Phi2, Phi3, Llama2, Llama2Chat, Mistral. Please refer to KAITO’s [readme](https://github.com/Azure/kaito/blob/main/presets/README.md) file for the latest models. 
+
+## Prerequisite
+1.	Before you begin, please make sure you have the following details from your infrastructure administrator:
+    - An AKS cluster that's up and running.

```

Comment:
Does this AKS Cluster mean an HCI cluster? Please define the term HCI cluster and use the term consistently across the document.

Reply:
changed to AKS Arc cluster, should be AKS arc anywhere, not scope to HCI.